tests/unit/mainwindow/test_prompt.py

```diff
@@ -80,8 +80,11 @@ def test_backspacing_path(self, qtbot, tmp_path, get_prompt):
         with qtbot.wait_signal(prompt._file_model.directoryLoaded):
             for _ in range(3):
                 qtbot.keyPress(prompt._lineedit, Qt.Key_Backspace)
+        prompt._set_fileview_root(prompt._lineedit.text())
 
-        # foo should get completed from f
+        # ...and foo should get completed from f
```

Comment:
```suggestion
        # '..' and 'foo' should get completed from 'f'
```

to make this clearer - i.e. the directory `..`, not the English "...and"